sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -240,6 +240,23 @@ public partial class CharFilter
         internal CharFilter() { }
         public string Name { get { throw null; } set { } }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct CharFilterName : System.IEquatable<Azure.Search.Documents.Indexes.Models.CharFilterName>
```

Comment:
Just a note: odd that this is newly added. @brjohnstmsft did the public swagger change for the GA version?

Reply:
But if nothing else of the API changed, this apparently isn't used and could be removed/hidden.

Reply:
Consequence of:

> Single value, modelAsString enums are generated as types, they were string constants before.